packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -49,6 +49,8 @@ export const withBurstLimit =
   async (execute, context) =>
   async (input) => {
     const config = context.limits
+    const customCapacitySet = getEnv('RATE_LIMIT_CAPACITY')
```

Comment:
We should also include a check on `RATE_LIMIT_CAPACITY_SECOND` and `RATE_LIMIT_CAPACITY_MINUTE` here too